src/components/Image/Image.tsx

```diff
@@ -6,6 +6,7 @@ import ImageElement from './ImageElement';
 type ImageProps = {
   alt: string;
   caption?: string;
+  captionDisplay?: string;
```

Comment:
You could declare explicit values here, because they are known (`below_image || hide || ...`). Allowing any string value could cause unexpected behaviour for somone who is trying to use the component (e.g. they provide captionDisplay = "belowImage"). 

If you declare the potential values explicitly, a type error will be thrown to let the person know the prop only accepts specific values.